webapp/channels/src/components/invitation_modal/invite_view.tsx

```diff
@@ -133,42 +132,37 @@ export default function InviteView(props: Props) {
 
     const errorProperties = {
         showError: false,
-        errorMessageId: '',
-        errorMessageDefault: '',
+        errorMessage: messages.exceededMaxBatch,
         errorMessageValues: {
-            text: '',
+            text: Constants.MAX_ADD_MEMBERS_BATCH.toString(),
         },
-        extraErrorText: '',
     };
 
     if (props.usersEmails.length > Constants.MAX_ADD_MEMBERS_BATCH) {
         errorProperties.showError = true;
-        errorProperties.errorMessageId = t(
-            'invitation_modal.invite_members.exceeded_max_add_members_batch',
-        );
-        errorProperties.errorMessageDefault = 'No more than **{text}** people can be invited at once';
-        errorProperties.errorMessageValues.text = Constants.MAX_ADD_MEMBERS_BATCH.toString();
     }
 
-    let placeholder = formatMessage({
-        id: 'invite_modal.add_invites',
-        defaultMessage: 'Enter a name or email address',
-    });
-    let noMatchMessageId = t(
-        'invitation_modal.members.users_emails_input.no_user_found_matching',
-    );
-    let noMatchMessageDefault =
-        'No one found matching **{text}**. Enter their email to invite them.';
-
-    if (!props.emailInvitationsEnabled) {
+    let placeholder;
+    let noMatchMessage;
+    if (props.emailInvitationsEnabled) {
+        placeholder = formatMessage({
+            id: 'invite_modal.add_invites',
+            defaultMessage: 'Enter a name or email address',
+        });
+        noMatchMessage = messages.noUserFound;
+    } else {
         placeholder = formatMessage({
             id: 'invitation_modal.members.search-and-add.placeholder-email-disabled',
             defaultMessage: 'Add members',
         });
-        noMatchMessageId = t(
-            'invitation_modal.members.users_emails_input.no_user_found_matching-email-disabled',
-        );
-        noMatchMessageDefault = 'No one found matching **{text}**';
+        noMatchMessage = messages.noUserFoundEmailDisabled;
+    }
+
+    let validAddressMessage;
```

Comment:
Many of these conditions could use the ternary operator (i.e. `?`). 0/5 on readability.

Reply:
This one could, but I'm going to leave it since I don't think it makes a major difference. For the other ones, I think the if-else is easier to read because I don't like multi-line ternary statements and I prefer to group things like
```
if (condition) {
    value1 = 'condition true';
    value2 = 'condition true';
} else {
    value1 = 'condition false';
    value2 = 'condition false';
}
```
instead of 
```
value1 = condition ? 'condition true' : 'condition false';
value2 = condition ? 'condition true' : 'condition false';
```
because it makes it immediately clear that the values are related by only checking the condition once